capstone/src/test/java/com/google/step/clients/RestaurantClientTest.java

```diff
@@ -112,18 +93,21 @@ public void testGetSingleRestaurant() {
 
     // Run client method
     RestaurantClient restaurantClient = new RestaurantClient();
-    Restaurant result = restaurantClient.getSingleRestaurant(restaurantKey);
+    Optional<Restaurant> result = restaurantClient.getSingleRestaurant(restaurantKey);
+    Restaurant actual = null;
+    if (result.isPresent()) {
+      actual = result.get();
+    } 
     assertNotNull("No response received", result);
 
     List<String> cuisineList = new ArrayList<>();
```

Comment:
`List<String> cuisineList = Arrays.asList("Steakhouse", "American");`

Reply:
Changed all of the lists--thanks!